contributed_definitions/nyaml/NXapm.yaml

```diff
@@ -559,2745 +392,557 @@ NXapm(NXobject):
-          and (primarily) imaging gas ions (in the case of field ion
-          microscopy) are accelerated towards a position-sensitive
-          and time-of-flight taking detector system.
-          Hence, there is no real probe/beam.
-        * Second, the specimen is the lens of the microscope.
-      (NXcsg):
-        exists: optional
-      instrument_name:
-        doc: |
-          Given name of the atom probe at the hosting institution. This is an
-          alias. Examples could be LEAP5000, Raptor, Oxcart, one atom at a time,
-          etc.
-      location:
-        exists: optional
-        doc: |
-          Location of the lab or place where the instrument is installed.
-          Using GEOREF is preferred.
-      (NXfabrication):
-        exists: recommended
-        vendor:
-          exists: recommended
-        model:
-          exists: recommended
-        identifier:
-          exists: recommended
-        capabilities:
-          exists: optional
-      flight_path_length(NX_FLOAT):
-        unit: NX_LENGTH
-        doc: |
-          The space inside the atom probe along which ions pass nominally
-          when they leave the specimen and travel to the detector.
-          
-          THIS DOCSTRING NEEDS CLARIFICATION.
-      
-      # NEW ISSUE: discussion depends on the type of instrument,
-      # straight flight path, curved, there were a few comments to made
-      # https://fairmat-experimental.github.io/nexus-fairmat-proposal/
-      # 50433d9039b3f33299bab338998acb5335cd8951/classes/
-      # contributed_definitions/NXapm.html
-      # where further details of the flight geometry should be recorded however
-      # in the majority of cases these data are not offered by APSuite and
-      # so far nobody has asked or seriously proceeded with how to use these
-      # pieces of information if they were to be stored
-      # MK::NEW ISSUE
-      field_of_view(NX_FLOAT):
-        exists: recommended
-        unit: NX_LENGTH
-        doc: |
-          The nominal diameter of the specimen ROI which is measured in the
-          experiment. It is important to mention that the physical specimen
-          cannot be measured completely because ions may launch but not be
-          detected or hit elsewhere in the analysis_chamber.
-      (NXreflectron):
-        
-        # check if doc string gets carried over doc: Device for reducing flight time differences of ions in ToF experiments.
-        applied(NX_BOOLEAN):
-          doc: |
-            Is a reflectron installed and was it used?
-        name:
+        To avoid these ambiguities, instances of :ref:`NXtransformations` should
+        be used.
+      (NXcoordinate_system):
+        exists: [min, 1, max, infty]
+        origin(NX_CHAR):
+        alias(NX_CHAR):
+        type(NX_CHAR):
+        handedness(NX_CHAR):
+        x_direction(NX_CHAR):
+        y_direction(NX_CHAR):
+        z_direction(NX_CHAR):
+
+    measurement(NXapm_msr):
```

Comment:
No because I do want to show people, like in EM btw right now that you can use the same appdef to describe sim and msr together in the same appdef. A computer simulation of a measurement is not a measurement. I see your point but also not convinced right now this is really a key issue here right now.
Assume you have a research study where you have done one msr and one sim trying to backup that one with the approach here chosen you can serialize this into one NeXus artifact with NXentry/msr and NXentry/sim.